doc/01-About.md

```diff
@@ -12,8 +12,28 @@ visualizing monitoring data in the Icinga ecosystem, consisting of:
   [Icinga DB Web](https://icinga.com/docs/icinga-db-web) module enabled,
   which connects to both Redis® and the database to display and work with the most up-to-date data
 
+## Big Picture
+
 ![Icinga DB Architecture](images/icingadb-architecture.png)
 
+Icinga DB consists of several components in an Icinga setup.
+This section tries to help understanding how these components relate, following the architecture diagram shown above.
+
+First things first, Icinga DB is not a database itself, but consumes and passes data from Icinga 2 to be displayed in Icinga DB Web and persisted in a relational database.
+
+Let's start with Icinga 2.
+With the Icinga DB feature enabled, Icinga 2 synchronizes its state to a Redis® server that can be queried by both the Icinga DB daemon and Icinga DB Web.
+
+The Icinga DB daemon consumes all the information from Redis®, transforms it, and finally inserts it into a relational database such as MySQL, MariaDB, or PostgreSQL.
```

Comment:
The word 'consumes' makes me think that it deletes the information from Redis as soon as it processes it. Is that the case? :)

Reply:
Technically, some information is being consumed in the way you have described it. However, the state information used by the Icinga DB Web module persists.

But I would maybe use a more vague word like "reads".

Reply:
I have slightly reworded this. Please take another look.

Reply:
LGTM